Sources/Internal/LayoutItemTypeEnumerator.swift

```diff
@@ -127,13 +132,17 @@ final class LayoutItemTypeEnumerator {
     case .day(let day):
       let nextDay = calendar.day(byAddingDays: 1, to: day)
       if day.month != nextDay.month {
-        return .monthHeader(nextDay.month)
+        return .monthFooter(day.month)
```

Comment:
Should this actually be combined with the `else if day == dayRange.upperBound` (which would indicate that we're at the last day in the total day-range of the calendar? If we don't return a monthFooter from that case, then we'd potentially miss showing a month footer at the end of the calendar.